docs/endpoints/getuids.md

```diff
@@ -0,0 +1,27 @@
+# Getting the existing user syncs
+
+This endpoint is used to get the existing user syncs.
+
+## `GET /getuids`
+
+This endpoint parses the PBS cookie and returns the existing UserIDs for each bidder.
+
```

Comment:
May want to mention if there is no cookie then they will get an empty return. Also a note that the purpose is for publishers to be able to pull the cookie sync info for a user to be used in a server to server request, where the originating server will not have access to the PBS user sync cookie.

Reply:
@hhhjort Thanks for the feedback. Updated the PR.